### ISSUE TITLE
2021-06-02 All-hands Catch-up

### ISSUE THREAD

**Issue by altsalt** [SeaGL/organization#126]
# 2021-06-02 SeaGL 2021 memorial day bi-weekly all-hands catch-up
- June 2nd, 2021. 1700 Pacific Time
- [Agenda](https://github.com/SeaGL/organization/issues/126)
- [Shared Pad](https://etherpad.seattlematrix.org/p/SeaGL_2021_all-hands)
- [Jitsi URL](https://meet.seattlematrix.org/SeaGL_2021_all-hands)
- [Previous meeting minutes](https://github.com/SeaGL/organization/blob/master/meetings/2021/)

### Check-in round
- Attendees: 
- Apologies: Salt
- Facilitator: 
- Note Taker(s): 
- Note Taker next meeting: Nathan

### Metrics (previous --> current)
- Keynote nominations: 12 --> 35
- (keep brainstorming!)

### Previous meeting feedback
- many kudos:
  - keith: great meeting, hope to be more around in next couple of weeks
  - andrew: seems to be a good meeting format
  - deb: great to see everyone, like going through tickets
  - norm: new to all of this, interest, will need to not use tablet for meetings
  - der.hans: good meeting, look forward to catching up
  - rachel: still receiving benefit of rules without them being as strict, meetings running much more smoothly
  - salt: thanks, want to work on integrating the issues a bit, would like to hear from people we haven't heard from for a while
- GH issue emails are a bit noisy (keith has offered help for anyone having this... issue)
- Comb GH issues a bit more beforehand to expedite things

## Minute of silence (get into the SeaGL headspace)
- Review agenda
- Consider tensions, progress updates, blockers, questions
- Gather personal notes, tasks, emails

# Agenda

### 3/8
- [ ] #75 transition fiscal sponsorship away from the FSF (Salt and Deb)
  - Salt: OSI has tentatively approved our request, needed a sustainability statement, Salt drafted it, still working on final approval
- [ ] #78 Document the new project management system (Salt)
  - Salt: underwater with finals week projects/grading, but it is on track for Mid-June
- [ ] #79 Finalize committee descriptions (Salt)
  - Salt: underwater with finals week projects/grading, but it is on track for Mid-June
- [ ] #80 Fill out SeaGL 2021 staff lead seats (Salt)
  - still looking for: 1x Finance, 1x Promotion/Outreach, 1x Partnerships, 2x Volunteers, 1x IDEA, 2x CoC
- [ ] #81 Set up asynchronous method to review feedback from last year (Salt)
  - Salt: still waiting for more responses, welcoming of any suggestions
- [ ] #82 Publish 2020 videos (Tech)
- [ ] #83 Ship 2020 speaker swag (Salt)
  - Salt: underwater with finals week projects/grading, but it is on track for Mid-June, bit blocked by #106

### 5/3
- [ ] #106 Talk to programming and partnerships about what to include in 2020 gift shipments (Salt)
  - Salt: Ideally, this will include update about talk recordings
- [ ] #110 Prepare 2021 Sponsor Prospectus (Finance and Partnerships)
  - Salt: Hans and Christy should be sure to sync

### 5/17
- [ ] #122 Send instructions for everyone to help with YouTube video transcription. (Keith)
  - Hans: will we have standalone transcripts as well?
  - Nathan: I haven't heard anything about this, but we should be careful, as not all talks can stand on their own with just a transcript and no slides
  - Salt: Dealing with some permission issues, but it sounds like folks will be able to login soon, a list of video links has been created and will go out for signups soon
- [x] #85 Add 2021 timeline to pad, ask for addition in GH issue (Salt)
- [ ] #56 Review [draft of virtual announcement blog post](https://github.com/SeaGL/seagl.github.io/pull/245) and LGTM (Rachel and Ben)
  - Salt: PR still open bit seems like it has been reviewed
  - Salt: lets update the post date since it's been a while, and get it published ASAP
- [x] #65 Add the note to template that "Meeting minute notes are public by default." (Salt)
- [x] #66 Review then migrate public issues from staff repo to org repo (Salt)
- [ ] #59 Finalize and send 2021 ads to Linux Magazine (Ben)
- [ ] #112 All committees have had their first meeting, and figured out whether a regularly scheduled one makes sense (Chairs)
- [x] Add Ben Cotton to the announce list moderators list (Salt)
- [x] Add a box for Fediverse contact on the keynote nomination form (in addition to existing Twitter) (Rachel)
  - Hans: Thank you for adding this new field!
- [x] #67 Share/retweet keynote nomination link (Everyone)
  - Nathan: Thanks everyone for your help in promoting this
  - Nathan: closing tonightish
- [ ] #68 Reach out to James Mason, lead of LFNW and OSEM maintainer (Andrew)
- [ ] #69 Put out a call for more tech volunteers, especially for OSEM/Rails on social media (Outreach)
- [x] #70 Start planning and announcing the 2021 career expo (der.hans)
  - Hans: announced to previous organizers, scheduling meetings/correspondance
- [ ] #71 Write email to SeaGL announce list (Outreach)

## New Items
- [ ] #123 Clean-up email ratsnest
- [ ] #124 Close out 2020, that list in the corner we never look at...
  - Salt: please review 2020 Wrapup tickets and take yours or pass them to someone else, so we can get these wrapped up

### IRC (Nathan)
- PR to update web site to change to Libera.chat seems like it was merged
- no announcement seems to have been made thus far
- Salt: blockers are having a blog post and a functioning bridge into Libera.chat from Matrix
- Salt: Matrix is having issues with some channels unbridging from Freenode
- Salt: we need to upgrade the Matrix channel version either way, so this seems like a good time to transition from Matrix.org to Seattle Matrix instance
- Nathan: shouldn't let Matrix/Blog Post block us, just put out a Tweet and migrate $now
- Salt: I have procedure in mind for how to handle shutting down channel, agree we can at least de-link for now

### Keynote Next Steps (Nathan)
- Numbers are looking much better, up to 35 now
- next steps: voting will be sent out to staff
- we are considering using civs rather than google form
- Salt: would you consider using https://star.vote ?
- Salt: ++ for using some form of ranked choice voting instead of a google form

### Lightning Talks (Nathan)
- we had a proposal for lighting talks
- Sri and Alison are going to head up the initiative
- should consider whether or not we can use specific branding for "auto-slide-progression talks"

### Team Drive (Nathan)
- programming committee votes against move off Google mid-planning
- can we enable this organizational feature of Google drive?
- Salt: sure!

### 5/31 - New TODOs
- [ ] queue up social media posts and unbridge Matrix channel from Freenode
- [ ] enable Team Drive for Google Org (Salt or Nathan)
- [ ] add transition to OSI banking as blocker for prospectus issue (Salt)
- [ ] send out draft of sponsor prospectus to seagl2021@ (Finance)
- [ ] select 2021 theme (Everyone)
  - [ ] add previous brainstorming to this issue (Salt)
- [x] verify Wednesday meeting time is 5pm Pacific (Salt)

### Timeline
- ~~May 10 (Monday)*: Launch Keynote request for nominations~~
- ~~May 24 (at the earliest): Close Keynote nominations~~
- June 15 (Tuesday)**: CFP opens
- August 5 (Thursday)*: CFP closes
- Week Of Aug 23 (choose w/ committee): CFP Review 1
- Week Of Aug 30 (choose w/ committee): Select Speakers
- September 6 (Monday)*: Notify Speakers
- September 19 (Monday): Finalize Speakers [this is the date the speakers need to have confirmed by, and that we'll need to have pulled waitlisted people up]
- September 26 (Monday)*: Finalize Program
- November 5-6 (Friday/Saturday)***: SeaGL 2021

`* means a blog post accompanies`  
`** there should be at least one additional blog post, asking for more specific things, halfway through the CFP being open`

### Next Meeting
- Catch up meeting on Wednesday on June 2nd, at 5pm PT (8pm ET)
  - Deb will facilitate for folks that can't make Monday's meeting
- Next regular bi-weekly all-hands Monday, June 14 @ 5pm PT

### Sign-off round (what's next, appreciations, suggestions)
- Nathan: in the heat of it for programming, nice to have a committee
- Norm: still looking for place to fit in and help out, preferably not as committee lead, wants to help with video transcriptions
- Hans: welcome/welcome back everyone, nice to hear the programming committee has new folks
- Ryan: going to dig through GitHub issues and see where I can help out
- Christy: looking forward to connecting offline with finance to work on prospectus and talk about sponsors, thanks to everyone for being involved
- Salt: <3

# Meeting adjourned!

**Comment by altsalt** [comment]
https://github.com/SeaGL/organization/blob/master/meetings/2021/20210602-allhands.md
